test/blackbox-tests/test-cases/action-runner/worker.t

```diff
@@ -0,0 +1,6 @@
+Connecting without server should fail
```

Comment:
Can we also have an end-to-end test where we use Dune with an action runner to build something?

Ideally, I'd like to see a test where we
* Start an action runner.
* Start a build that uses this action runner to run some of the actions.
* The build completes successfully and we can confirm that the action runner was used to execute the actions.